src/main/java/com/example/WMS/services/servicesimpl/OrderServicesImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.example.WMS.services.servicesimpl;
+
+import com.example.WMS.dtos.AddStockDescriptionDto;
+import com.example.WMS.dtos.OrderDescriptionDto;
+import com.example.WMS.dtos.OrderDto;
+import com.example.WMS.dtos.ValidationDto;
+import com.example.WMS.entity.Order;
+import com.example.WMS.entity.ProductDetails;
+import com.example.WMS.handlers.OrderHandler;
+import com.example.WMS.handlers.ProductDetailsHandler;
+import com.example.WMS.handlers.ReadWriteHandler;
+import com.example.WMS.handlers.ReceiveOrdersHandler;
+import com.example.WMS.services.OrderServices;
+import org.springframework.stereotype.Service;
+
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class OrderServicesImpl implements OrderServices {
+    public ValidationDto createOrder(OrderDto orderDto) {
+
+        ProductDetails productDetails = ProductDetailsHandler.getProductDetails();
+//        Thread thread = new Thread(new ReadThread(productReader, "read"));
+//        thread.start();
+//        try{
+//            thread.join();
+//        }
+//        catch (Exception e) {
+//            System.out.println(e);
+//        }
+        System.out.println(productDetails.toString());
+        if(productDetails.getQuantity() < orderDto.getQuantity()) {
```

Comment:
It can result in Null Pointer Exception since productDetails can be returned as null from above function

Reply:
handling errors better